import/repos.json

```diff
@@ -18,7 +18,7 @@
       },
       {
         "version": "v5",
-        "name": "release/oss-v5"
+        "name": "v5_docs_folder_structure_alginment"
```

Comment:
this will be updated (before merging) back to a release branch once the database PR is merged